os/kernel/sched/sched_releasetcb.c

```diff
@@ -211,6 +214,18 @@ int sched_releasetcb(FAR struct tcb_s *tcb, uint8_t ttype)
 
 		ret = up_addrenv_detach(tcb->group, tcb);
 #endif
+#if defined(CONFIG_APP_BINARY_SEPARATION) && defined(CONFIG_ARM_MPU)
+		/* Disable mpu regions when binary is unloaded */
+		if (IS_LOADED_MODULE(tcb->group)) {
```

Comment:
Is this correct? Can we check the loaded module with `tcb->group`? not `tginfo`?